reposilite-site/data/guides/integrations/caddy.md

```diff
@@ -0,0 +1,58 @@
+---
+id: caddy
+title: Caddy
+---
+
+Caddy is a http server with support for reverse proxy and automatic https. The automatic https feature 
+takes care of requesting, installing and updating SSL certificates which means that you need much less configuration settings
+or maintenance compared with e.g. Nginx.
+
+Note that websockets are also transparent in the case of reverse proxying.
+
+First [download and install caddy](https://caddyserver.com/docs/install) and modify your caddy configuration file, 
+typically found in `/etc/caddy/Caddyfile`.
+
+```json5
+
+(revproxy) {
+         #compress responses
+         encode zstd gzip
+
+         #redirect from http to https
+         @http {
+                protocol http
+         }
+         redir @http https://{host}{uri}
+
+         reverse_proxy localhost:{args.0}
+         file_server
+}
+
+domain.com {
+        root * /opt/reposilite/static
+        import revproxy 8080
+}
+```
+
+You might need to modify a few things:
+
+* If Reposilite is running on an other machine, replace `localhost` in the `reverse_proxy localhost:{args.0}` statement to the IP or host of that machine.
+* Here the `domain.com` refers to the domain name in used. Typically this needs to be changed.
+* The port of the Reposilite instance here is 8080, this needs to match your configuration.
+* With URL-rewriting it is also possible to redirect a directory `/reposilite' to your instance. I leave this as an excercise for the reader.
```

Comment:
I'd replace `I leave this as an excercise for the reader.` with a link to Caddy's guide about that 😅